### ISSUE TITLE
[CHANGE] There should not be a distinction between internal and external

### ISSUE THREAD

**Issue by jrowsell** [gc-da11yn/gc-da11yn.github.io#221]
## Required information

**[What page is the change request for? Please supply the URI.](https://a11y.canada.ca/en/standards/standard/)**

1.4 Public-facing web content
1.4.1 Subsections [2.3.2.2](https://a11y.canada.ca/en/standards/standard/#2.3.2.2) and [2.3.2.3](https://a11y.canada.ca/en/standards/standard/#2.3.2.3) must conform with this standard by (effective date + 12 months)

1.7 Internal-facing web content
1.7.1 Subsection [2.3.2.4](https://a11y.canada.ca/en/standards/standard/#2.3.2.4) must conform with this standard by (effective date + 24 months)

Separating internal and external facing digital content and ICT is a disservice to those currently experiencing barriers. We have inaccessible time management systems, pay systems, leave systems, intranet and constantly created inaccessible documents being created and shared internally and externally. The narrative that internal services, products and content need more time is poor stewardship. It's been a known issue for more than 15 years. There are official channels of complaints around this and requires stronger pushes to be achieved. This separate further articulates to people that they may view internal and external digital content differently with different prioritization mechanisms, levers, support structures and weaker decision-making practices.

**Comment by mgifford** [comment]
Yes, but I don't think I've found a Government of Canada site that fully meets WCAG 2.0 AA externally. Many of the sites using Drupal will be pretty close to meeting WCAG 2.0 AA for authoring purposes (internal). 

De-prioritization of accessibility in the authoring (and admin) sides of digital tools for decades has produced considerable technical debt. Reworking the authoring interface of some of the custom-built, proprietary or legacy tools government still uses may be next to impossible. A timeline is important, but I don't think that even with the proposed longer timeline (and a lot of resources) that the government will be able to achieve this. 

Systems will need to be decommissioned and replaced with tools (like Drupal) which have prioritized accessibility. Accessibility is like a blueberry muffin after all, kinda doesn't work to add in the blueberries after you have baked the muffins. 

I'd like them to be the same, but know that they are tasks with substantially different level of complexity. It is easier to start with the external sites (which just continues the problem). 

One big advantage of doing them both at once is that it might be determined up-front that fixing the back-end is going to be impossible (or nearly so). It might encourage departments to abandon projects earlier that will not be able to achieve the targets of EN 301 549 compliance (let alone WCAG 2.0 AA). Is it better to rip the band-aid off quickly or slowly? It is going to hurt a lot either way.
